components/Layout/HomeHeader/HomeHeaderEpisodeTitle.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div>
+    <p class="text-overline text-truncate mb-2 my-2">
+      {{ $t('homeHeader.items.recentlyAdded') }}
+    </p>
+    <v-img
+      v-if="item.ParentLogoImageTag || (item.ImageTags && item.ImageTags.Logo)"
```

Comment:
What about computing this using imageHelper's mixin so it's more concise and clearer?